test/index.test.ts

```diff
@@ -58,6 +80,86 @@ describe("My Probot app", () => {
     await probot.receive({ name: "issue_comment", payload });
   });
 
+  test("creates a comment with pick-up command to a pull request", async () => {
+    nock("https://api.github.com")
+      .post("/app/installations/2/access_tokens")
+      .reply(200, { token: "test" })
+      // Notice: Actually, it returns the pull request information here.
+      .get(`/repos/Rustin-Liu/ti-challenge-bot/issues/1`)
+      .reply(200, {
+        id: 1,
+        number: 1,
+        state: "open",
+        body: "I'm having a problem with this.",
+        labels: [],
+        user: {
+          login: "Rustin-Liu",
+          id: 1,
+        },
+        pull_request: {},
+      })
+      .get(
+        "/repos/Rustin-Liu/ti-challenge-bot/contents/.github/challenge-bot.yml"
+      )
+      .reply(200, {
+        type: "file",
+        encoding: "base64",
+        size: 5362,
+        name: "challenge-bot.yml",
+        path: ".github/challenge-bot.yml",
+        content: "",
+        sha: "3d21ec53a331a6f037a91c368710b99387d012c1",
+        url:
+          "https://api.github.com/repos/Rustin-Liu/ti-challenge-bot/contents/.github/challenge-bot.yml",
+      });
+
+    // Receive a webhook event.
+    await probot.receive({ name: "issue_comment", payload: pickUpPayload });
+
+    // If the robot receive a pick-up command to a pull request, the `pickUp` method will not be called.
+    expect(mockPickUpMethod.mock.calls[0]).toBeUndefined();
+  });
+
+  test("creates a comment with give-up command to a pull request", async () => {
+    nock("https://api.github.com")
+      .post("/app/installations/2/access_tokens")
+      .reply(200, { token: "test" })
+      // Notice: Actually, it returns the pull request information here.
+      .get(`/repos/Rustin-Liu/ti-challenge-bot/issues/1`)
+      .reply(200, {
+        id: 1,
+        number: 1,
+        state: "open",
+        body: "I'm having a problem with this.",
+        labels: [],
+        user: {
+          login: "Rustin-Liu",
+          id: 1,
+        },
+        pull_request: {},
+      })
+      .get(
+        "/repos/Rustin-Liu/ti-challenge-bot/contents/.github/challenge-bot.yml"
+      )
+      .reply(200, {
+        type: "file",
+        encoding: "base64",
+        size: 5362,
+        name: "challenge-bot.yml",
+        path: ".github/challenge-bot.yml",
+        content: "",
+        sha: "3d21ec53a331a6f037a91c368710b99387d012c1",
+        url:
+          "https://api.github.com/repos/Rustin-Liu/ti-challenge-bot/contents/.github/challenge-bot.yml",
+      });
+
+    // Receive a webhook event.
+    await probot.receive({ name: "issue_comment", payload: giveUpPayload });
+
+    // If the robot receive a give-up command to a pull request, the `giveUp` method will not be called.
+    expect(mockGiveUpMethod.mock.calls[0]).toBeUndefined();
```

Comment:
It can be `expect(mockGiveUpMethod).not.toHaveBeenCalled()`.

Reply:
Good idea!